src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -3176,6 +3201,9 @@ because xbuild doesn't support framework reference assemblies.
   </CalculateProjectDependencies>
 </Target>
 
+<!-- Our first attempt to split up this file -->
+<Import Project="$(MSBuildThisFileDirectory)Xamarin.Android.D8.targets" />
```

Comment:
@jonathanpeppers should be import be above the first instance of `_CompileToDalvikWithD8 ` ? (line 2649)

Reply:
The problem here is msbuild will error if it cannot find the `_CompileToDalvikWithD8` target. I think a conditional import is probably not needed. I would just move it up. We should probably define an area for all the new "imports" we are going to do which is under the various property blocks but before the bulk of the targets I guess.